playwright-tests/tests/stake-delegation/stake-delegation.spec.js

```diff
@@ -984,7 +983,9 @@ test.describe("Lockup staking", function () {
     }) => {
       test.setTimeout(120_000);
       await openLockupStakingForm({ page, daoAccount, lockupContract });
+      await page.waitForTimeout(20_000);
       const poolSelector = page.locator(".custom-select").first();
+      await expect(poolSelector).toBeVisible({ timeout: 20_000 });
```

Comment:
Also here I am wondering what issue this is related to. Is it testing a bug fix that for a bug that is not registered?

Reply:
it's because of flakiness, the `disabled` class was not visible, so I added this to make sure the dropdown is visible first and then it checks for` disabled` class